### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by AnnaSalata**

### CODE REVIEW

File: logger.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const config = require('./config');
+
+const createMessage = () => {
```
Comment (franxx32) [inline logger.js]: Logger should log not only one message
It should be an object with different method for logs
For instance: logger.info(msg), logger.error(message)
Take a look at https://github.com/pinojs/pino

File: app.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const http = require('http');
+const config = require('./config');
+const logger = require('./logger');
+
+http.createServer((req, res) => {
+    console.log('New incoming request');
```
Comment (franxx32) [inline app.js]: Need to use logger here -> logger.info('New incoming request')

File: app.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    res.writeHeader(200, { 'Content-Type': 'application/json' });
+    res.end(JSON.stringify({ message: 'Hello world!'}));
```
Comment (franxx32) [inline app.js:8]: logger.info(Server is listening on port ${config.APP_PORT}. Env is ${config.ENV})

File: logger.js (hunk 1 above)
Comment (AnnaSalata) [reply logger.js]: fixed

File: app.js (hunk 2 above)
Comment (AnnaSalata) [reply app.js]: fixed

File: app.js (hunk 3 above)
Comment (AnnaSalata) [reply app.js:8]: fixed

**Review by franxx32 — APPROVED** [review]

**Pull request merged**